internal/aircraftlib/aircraft.capnp.go

```diff
@@ -2245,11 +2245,12 @@ func (s Z) SetAnyList(v capnp.Ptr) error {
 	return s.Struct.SetPtr(0, v)
 }
 
-func (s Z) AnyCapability() (capnp.Ptr, error) {
+func (s Z) AnyCapability() *capnp.Client {
```

Comment:
I like that this patch removes a case where you have to check an error.

Reply:
Yeah, same!  In fact, I was just wondering if we could similarly ignore errors in `SetAnyCapability`?  I'm guessing _not_, since pointers require jumping to a different offset in the segment, which may or may not exist.  Is that right?  Or is there another reason as well?

Reply:
I think so, and in fact we could do the same for all interface types. Other pointer types could fail because they might need to allocate a landing pad for a far pointer, and the allocation could fail, but cap pointers don't ever need to do this so this can't actually fail.